docs/source/tutorial.ipynb

```diff
@@ -303,7 +473,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.6.15"
```

Comment:
I have put down a 3.6.x python version as we support 3.6.x as what I learned from @andre-merzky 's comments in the chat history somewhere.